src/Expensify.tsx

```diff
@@ -184,6 +185,10 @@ function Expensify() {
     useEffect(() => {
         // Initialize Fullstory lib
         FS.init(userMetadata);
+        FS.onReady().then(async () => {
+            const session = await FS.getSessionId();
+            Sentry.setTag('fullstorySession', session);
```

Comment:
not sure what the convention is for tags, but we might want to follow https://docs.sentry.io/platforms/javascript/enriching-events/tags/ and go with `fullstory_session_id`.